src/diffusers/models/transformers/auraflow_transformer_2d.py

```diff
@@ -232,7 +233,7 @@ def forward(
         return encoder_hidden_states, hidden_states
 
 
-class AuraFlowTransformer2DModel(ModelMixin, ConfigMixin):
+class AuraFlowTransformer2DModel(ModelMixin, ConfigMixin, PeftAdapterMixin, FromOriginalModelMixin):
```

Comment:
I don't think we need `FromOriginalModelMixin`. No?

Reply:
Nope, Removed.